src/Microsoft.Azure.WebJobs.Host/Executors/DefaultStorageAccountProvider.cs

```diff
@@ -15,6 +16,7 @@ internal class DefaultStorageAccountProvider : IStorageAccountProvider
         private readonly IStorageCredentialsValidator _storageCredentialsValidator;
         private readonly IStorageAccountParser _storageAccountParser;
         private readonly IServiceProvider _services;
+        private readonly ConcurrentDictionary<string, Task<IStorageAccount>> _accounts = new ConcurrentDictionary<string, Task<IStorageAccount>>();
```

Comment:
Why is this a dictionary mapping strings to tasks? It should be strings to IStorageAccount.